src/components/access-type.js

```diff
@@ -1,230 +1,310 @@
-import {connect, batch} from 'react-redux';
+import React from "react";
+import { connect, batch } from "react-redux";
 import get from "lodash/get";
 import {
-    ACCESS_BEING_LOADED,
-    ACCESS_UPDATED,
-    PAYMENT_OPTIONS_UPDATED,
-    SUBSCRIPTION_GROUP_UPDATED,
-    METER_UPDATED
+  ACCESS_BEING_LOADED,
+  ACCESS_UPDATED,
+  PAYMENT_OPTIONS_UPDATED,
+  SUBSCRIPTION_GROUP_UPDATED,
+  METER_UPDATED,
+  ASSET_PLANS
 } from "../store/actions";
 import PropTypes from "prop-types";
-import {awaitHelper} from "../utils";
+import { awaitHelper } from "../utils";
 
 class AccessTypeBase extends React.Component {
+  componentDidMount() {
+    this.initAccessType();
+  }
 
-    componentDidMount() {
-        this.initAccessType();
-    }
-
-    loadScript = callback => {
-        const accessTypeKey = get(this.props, ['accessTypeKey']);
-        const isStaging = get(this.props, ['isStaging']);
-        const enableAccesstype = get(this.props, ['enableAccesstype']);
-
-        if(!enableAccesstype){
-            return false;
-        }
-
-        if(accessTypeKey && !global.AccessType && global.document) {
-            const accessTypeScript = document.createElement('script');
-            let accessTypeHost = `https://www.accesstype.com/frontend/accesstype.js?key=${accessTypeKey}`;
-            if(isStaging){
-                accessTypeHost = `https://staging.accesstype.com/frontend/accesstype.js?key=${accessTypeKey}&env=sandbox`;
-            }
-            accessTypeScript.setAttribute("src", accessTypeHost);
-            accessTypeScript.setAttribute("data-accessType-script", "1");
-            accessTypeScript.async = 1;
-            accessTypeScript.onload = () => callback();
-            document.body.appendChild(accessTypeScript);
-            return true;
-        }
-
-        global.AccessType && callback();
-        return true;
-    };
+  loadScript = callback => {
+    const accessTypeKey = get(this.props, ["accessTypeKey"]);
+    const isStaging = get(this.props, ["isStaging"]);
+    const enableAccesstype = get(this.props, ["enableAccesstype"]);
 
-    setUser = async (emailAddress, mobileNumber) => {
-        if(!global.AccessType){
-            return {};
-        }
+    if (!enableAccesstype) {
+      return false;
+    }
 
-        const { error, data: user }  = await awaitHelper(global.AccessType.setUser({ 'emailAddress': emailAddress, 'mobileNumber':  mobileNumber}));
-        if(error) {
-            console.warn(`User context setting failed --> `, error);
-            return error;
-        }
-        return user;
-    };
+    if (accessTypeKey && !global.AccessType && global.document) {
+      const accessTypeScript = document.createElement("script");
+      let accessTypeHost = `https://www.accesstype.com/frontend/accesstype.js?key=${accessTypeKey}`;
+      if (isStaging) {
+        accessTypeHost = `https://staging.accesstype.com/frontend/accesstype.js?key=${accessTypeKey}&env=sandbox`;
+      }
```

Comment:
Suggestion: only domain(`staging.accesstype.com`) and `&env=sandbox` can be change conditionally instead of putting whole script link. 

if `accessTypeKey` is prop then it is better to get domain from props only.